client/new-stack-editor/lib/views/wizard/providerselectionview.coffee

```diff
@@ -0,0 +1,67 @@
+kd = require 'kd'
+JView = require 'app/jview'
+globals = require 'globals'
+Tracker = require 'app/util/tracker'
+
+
+module.exports = class ProviderSelectionView extends JView
+
+  constructor: (options = {}, data) ->
+
+    options.cssClass ?= 'provider-selection'
+
+    super options, data
+
+    @createProviders()
+
+
+  createProviders: ->
+
+    { providers } = globals.config
+
+    supportedProviders = (Object.keys providers).filter (p) ->
+      return p  if providers[p].supported
```

Comment:
```coffee
supportedProviders = (Object.keys providers).filter (p) -> providers[p].supported
```

actually with lodash there is an easier and -IMO- more readable way ([link to doc](https://lodash.com/docs/4.17.3#pickBy)):

```coffee
_.pickBy providers, (provider) -> provider.supported
```

All three of them do the same thing.